packages/color/src/Color.ts

```diff
@@ -482,6 +482,7 @@ export class Color
      * Rounds the specified color according to the step. This action is destructive, and will
      * override the previous `value` property to be `null`. The alpha component is not rounded.
      * @param steps - Number of steps which will be used as a cap when rounding colors
+     * @deprecated since 7.3.0
      */
     round(steps: number): this
     {
```

Comment:
Do we need a deprecation warning here?

Reply:
The deprecation function is in utils which would make a circular dependency between color and utils. I didn't want to change the package dependencies just for this, it's fine without the warning. No one is using `round` anyways.